andela/travel_tool_front
# #160813365 Fix Updated trips Table

#### What does this PR do?
This PR fixes errors that occur on the trip table after a successful update and also enables a change from a return trip to a one-way trip.

#### Description of Task to be completed?
- Update trips table after a successful edit
- Enable update from a return trip to a one-way trip
- Error messages should show in red
#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout bg-fix-updated-table-160956745`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Navigate to `localhost:3000` in a browser
- Create a return trip
- Edit your newly created trip and change it to a one-way trip.
- Also, check the updated trip to see that the table is immediately updated without a reload.

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[160956745](https://www.pivotaltracker.com/story/show/160813365)

#### Screenshots (if appropriate)
<img width="1054" alt="screenshot 2018-10-09 at 12 07 10 pm" src="https://user-images.githubusercontent.com/22764463/46665474-e10f4f80-cbbb-11e8-9a18-ca5315ad1131.png">


#### Questions:
